Source/WebCore/Scripts/tests/ProcessSyncClient.cpp

```diff
@@ -27,6 +27,7 @@
 #include "ProcessSyncClient.h"
 
 #include "ProcessSyncData.h"
+#include <wtf/EnumTraits.h>
```

Comment:
Is this needed?

Reply:
Not yet, this leaked through from the first patch I was working on yesterday...  Will remove for now.

Reply:
(NM, yes, this is required now)